e2e/terraform/outputs.tf

```diff
@@ -54,6 +54,9 @@ export NOMAD_CLIENT_CERT=${abspath(path.root)}/keys/tls_api_client.crt
 export NOMAD_CLIENT_KEY=${abspath(path.root)}/keys/tls_api_client.key
 export NOMAD_TOKEN=${data.local_sensitive_file.nomad_token.content}
 export NOMAD_E2E=1
+export CONSUL_HTTP_ADDR=https://${aws_instance.consul_server.public_ip}:8501
+export CONSUL_HTTP_TOKEN=${local_sensitive_file.consul_initial_management_token.content}
+export CONSUL_CACERT=${abspath(path.root)}/keys/tls_ca.crt
```

Comment:
this reminds me - care to add something like this too? https://github.com/hashicorp/nomad/commit/c7d01705f51944cd20f2d253e4d34d5f1f2518eb

for occasions when we ssh to a cluster built by CI (and therefore do not have access to TF outputs to see the token)

Reply:
Will do!